systemtest/src/test/java/io/strimzi/systemtest/CruiseControlST.java

```diff
@@ -35,20 +255,14 @@ void setup() {
         applyRoleBindings(NAMESPACE);
         // 050-Deployment
         KubernetesResource.clusterOperator(NAMESPACE).done();
-    }
 
-    @Test
-    void testCCDeployment()  {
         KafkaResource.kafkaWithCruiseControl(CLUSTER_NAME, 3, 3).done();
+    }
 
-        String ccStatusCommand = "curl -X GET localhost:9090/kafkacruisecontrol/state";
-        String ccPodName = PodUtils.getFirstPodNameContaining("cruise-control");
-        LOGGER.info("Using pod: " + ccPodName);
-        String result = cmdKubeClient().execInPodContainer(ccPodName, "cruise-control",
-                "/bin/bash", "-c", ccStatusCommand).out();
+    @Override
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces) throws InterruptedException {
+        super.recreateTestEnv(coNamespace, bindingsNamespaces);
 
-        assertThat(result, not(containsString("404")));
-        assertThat(result, containsString("RUNNING"));
+        KafkaResource.kafkaWithCruiseControl(CLUSTER_NAME, 3, 3).done();
```

Comment:
Guess you can create method for `deployTestResources` and use it `beforeAll` and in `recreate`